setup.py

```diff
@@ -19,6 +19,7 @@
         'sonic_platform_base.sonic_ssd',
         'sonic_psu',
         'sonic_sfp',
+        'sonic_platform_base.sonic_thermal_control'
```

Comment:
Please move this line below `sonic_platform_base.sonic_ssd` to retain alphabetical order